tests/src/ExistingSite/PersonTest.php

```diff
@@ -86,12 +88,16 @@ public function testCreatePerson() {
     $assert_session->pageTextNotContains('This field has been disabled because you do not have sufficient permissions to edit it.');
     $page->fillField('First name', 'Stefan');
     $page->fillField('Last name', 'Mayer');
-    $page->fillField('Short description', 'Example short description field.');
+    $field = $page->findField('Short description');
+    $this->assertFieldHasFormat($field, 'simple_rich_text');
+    $field->setValue('Example short description field.');
     $page->fillField('Use existing media', $media_image->getName() . ' (' . $media_image->id() . ')');
     $page->fillField('Country', 'DE');
     $page->fillField('Occupation', 'DG TEST');
     $page->fillField('Position', 'Director');
-    $page->fillField('Additional information', 'Example additional information field.');
+    $field = $page->findField('Additional information');
+    $this->assertFieldHasWysiwyg($field, 'rich_text');
```

Comment:
Basically with the comments below/above this becomes
```suggestion
    $this->assertEquals('rich_text', $this->getWysiwigTextFormat($field));
```